src/plugins/initiatives/config.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import * as C from "../../util/combo";
+import {_validateUrl} from "./initiativesDirectory";
+
+export type InitiativesConfig = {|
+  +remoteUrl: string,
+|};
+
+export function parseConfig(raw: C.JsonObject): InitiativesConfig {
+  return configParser.parseOrThrow(raw);
+}
```

Comment:
Don't need this. Just use `loadJson` or `loadJsonWithDefault`, which takes the parser directly.